ztoc/toc_builder.go

```diff
@@ -150,7 +150,7 @@ func metadataFromTarReader(r io.Reader) ([]FileMetadata, compression.Offset, err
 			ModTime:            hdr.ModTime,
 			Devmajor:           hdr.Devmajor,
 			Devminor:           hdr.Devminor,
-			Xattrs:             hdr.PAXRecords,
+			PAXHeaders:         hdr.PAXRecords,
```

Comment:
This wouldn't affect backwards content addressability, right?

Reply:
Yeah. It's backwards compatible with serialized ztocs because it doesn't change the serialization. The library did have an API change that we might want to document, though.